ts/client/src/accounts/mangoAccount.ts

```diff
@@ -1793,15 +1793,29 @@ export class PerpOoDto {
   ) {}
 }
 
-export class TokenConditionalSwapDisplayPriceStyle {
-  static sellTokenPerBuyToken = { sellTokenPerBuyToken: {} };
-  static buyTokenPerSellToken = { buyTokenPerSellToken: {} };
+export type TokenConditionalSwapDisplayPriceStyle =
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  | { sellTokenPerBuyToken: {} }
```

Comment:
I suggest using the recommended `Record<string, none>` over `{}` here so we can get rid of the lint-comment